admin/templates/base.html

```diff
@@ -82,6 +87,9 @@
             {% if 'prereg_group' in request.user.group_names %}
             <li><a href="{% url 'pre_reg:prereg' %}"><i class='fa fa-link'></i> <span>OSF Prereg</span></a></li>
             {% endif %}
+            {% if request.user.is_staff %}
+            <li><a href="{% url 'auth:register' %}"><span>User Registration</span></a></li>
```

Comment:
I would suggest pre-pending `Admin` onto the `User Registration` in order to make it clear that it isn't the OSF. Also `fa-link`